src/components/ColorPallete/colors.css

```diff
@@ -0,0 +1,30 @@
+:root {
+  --note-color-1: #efcb68;
+  --note-color-2: #e1efe6;
+  --note-color-3: #caffd0;
+  --note-color-4: #a9a587;
+  --note-color-5: #fbd1a2;
+  --note-color-6: #7dcfb6;
+  --note-color-7: #e5a4cb;
```

Comment:
could add which color it exactly is, if you want to replace a color with something else in future, you'll have to manually check which code is of which color